opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "anomalyDetectionDashboards",
-  "version": "1.3.8.0",
+  "version": "1.3.9.0",
```

Comment:
You missed two places to bump versions. Please check https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/commit/f424e2a3982c8dd3d78a45938c86df4679b333e3 for an example.

Reply:
Hey @kaituo, Sorry I might have accidentally missed to bump the versions, updated the changes. Thanks for reviewing.